src/pages/ESNMembersTab/EsnersSheet.tsx

```diff
@@ -15,12 +20,26 @@ const createEsner = (rawEntry: string[]) => {
         id: parseInt(rawEntry[0]),
         name: rawEntry[1],
         surname: rawEntry[2],
-        isBoard: Boolean(rawEntry[3]),
+        memberType: rawEntry[3],
         position: rawEntry[4],
         picture: rawEntry[5],
     };
 };
 
+function showMembers(data: ESNer) {
```

Comment:
shouldn't it be `showMember` ? 

As I understand properly it's rendering only one member

Reply:
Btw. You can transform it to react component that takes `data` as prop. 

Thus react will treat it differently/in more official way (I cannot find the article I wanted to post here now xd)

Reply:
done